frontend/.editorconfig

```diff
@@ -0,0 +1,15 @@
+  
+# http://editorconfig.org
```

Comment:
❤️ I didn't know about this. I think we could actually move this up to the root of the repo. All of this applies to Rails too

Reply:
Yup I can. However, I haven't used this, and got it from the linked codebase from airbnb's linting/prettier rules. I guess it will work if it's in the root and not `/frontend`?  If not, I guess that can be addressed in #50